modules/index/src/main/java/org/apache/ignite/internal/index/IndexChooser.java

```diff
@@ -139,16 +146,17 @@ public void close() {
     }
 
     /**
-     * Collects a list of table indexes that will need to be used for an update operation in RW a transaction. List consists of all indexes
-     * (available and registered) on the requested catalog version, as well as all dropped available indexes from previous catalog versions.
+     * Collects a list of table indexes that will need to be used for an update operation in an RW transaction. The list consists of all
+     * indexes (available and registered) on the requested catalog version, as well as all dropped available indexes from previous catalog
+     * versions.
      *
-     * <p>Returned list is sorted by {@link CatalogObjectDescriptor#id()}. It is expected that the table exists at the time the method is
-     * called.</p>
+     * <p>Returned list is sorted by {@link CatalogObjectDescriptor#id()}. The table is expected to exist in the catalog at the requested
+     * version.</p>
      *
      * @param catalogVersion Catalog version.
      * @param tableId Table ID.
      */
-    List<CatalogIndexDescriptor> collectForRwTxOperation(int catalogVersion, int tableId) {
+    List<CatalogIndexDescriptor> collectForRwTxUpdateOperation(int catalogVersion, int tableId) {
```

Comment:
In the class name, the terminology has changed from 'collector' to 'chooser'. Should 'collect' be changed to 'choose' as well?

Reply:
fix it